src/compas_ghpython/artists/frameartist.py

```diff
@@ -49,17 +49,32 @@ def draw(self):
 
         Returns
         -------
-        geometry : list
+        :class:`Rhino.Geometry.Plane`
 
-            * geometry[0] : :class:`Rhino.Geometry.Point`
-            * geometry[1] : list of :class:`Rhino.Geometry.Line`
+        """
+        return compas_ghpython.draw_frame(self.primitive)
+
+    def draw_origin(self):
+        """Draw the frame's origin.
+
+        Returns
+        -------
+        :class:`Rhino.Geometry.Point`
+
+        """
+        point, _ = self._get_args(self.primitive, self.scale, self.color_origin, self.color_xaxis, self.color_yaxis, self.color_zaxis)
+        return compas_ghpython.draw_points([point])
```

Comment:
this will return a list of Rhino points and not a single value as suggested in the docstring

Reply:
So do you prefer that the docstring is changed or the return value?

Reply:
i would vote for the return value